WcaOnRails/app/jobs/dump_developer_database.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class DumpDeveloperDatabase < ApplicationJob
+  queue_as :default
+
+  def perform
+    # Create developer database dump every 3 days.
+    last_developer_db_dump = Timestamp.find_or_create_by(name: 'last_developer_db_dump')
+    if last_developer_db_dump.date.nil? || last_developer_db_dump.date < 3.days.ago
```

Comment:
Darn, I was wrong on 2 out of 2 technical things...

There must be a way to set a default date in the schema, but I don't know it and your function is a good alternative. I like that it's in Ruby code rather than the harder to know schema. The danger is that if you don't use the function, you can get errors if you don't write tests. But that's always true...

But I think it needs a less awkward name. How about `not_after?()` or possibly `not_since?`? This flips the logic so we don't have to talk about `nil`.